lib/diagnostic.js

```diff
@@ -1,28 +1,46 @@
 'use strict'
+// Comfort: 3.5
+// Clarity: 4
+
+// I used prior class documents, Google, induction, and my memory to help with this assessment.*/
+
 
 // Reuse functions wherever possible.
 
 // 1. Write a function that takes a string argument
 // splits it into an array of normalized words
 // (uppercase strings without punctuation)
 // and returns that array.
-const normalizeWords = function (/* */) {
-  /* your response here */
+
+// "Muh muh muh muh muh muh meow"
+
+let lisaString = "skdj*#&$#*($*&#*$&#sdkfjdkfdjsdwekfekfjekfje   welrkwjekrjwlsk*$&*#$&{}#*woeifjeifje#&$alkdjfaksdjfkdsjfadfj)"
+
+const normalizeWords = function (paragraph) {
+  let myArray = paragraph.split(" ")
+  let myUpperArray = myArray.toUpperCase()
+  let myCleanArray = myUpperArray.replace(/[.,\/#!$%\^&\*;:{}=\-_`~()]/g,"")
+  return myCleanArray
 }
 
 // 2. Write a function that takes a string argument
 // and returns an array of unique normalized words.
-const uniqueWords = function (/* */) {
-  /* your response here */
+const uniqueWords = function (myCleanArray) {
+  let uniqueArray = []
+  for (let i = 0; i < myCleanArray.length; i++){
+    uniqueArray[i] = myCleanArray[i].count()
```

Comment:
You're on the right track here, but I don't think .count() will work the way you want it to.